include/hpp/rbprm/interpolation/time-constraint-helper.inl

```diff
@@ -248,7 +249,8 @@ namespace
                 core::segment_t interval(0, completePath->initial().rows()-1);
                 core::segments_t intervals;
                 intervals.push_back(interval);
-                PathPtr_t reducedPath = core::SubchainPath::create(completePath,intervals);
+                core::segments_t velIntervals (1, core::segment_t (0, device->numberDof()));
```

Comment:
In your case, I don't think it is important because you never call path->derivative, but shouldn't this be
`core::segments_t velIntervals (1, core::segment_t (0, device->numberDof()-1));`

Reply:
if you say so :). Thanks !